hiera/hosts/vagrant-test.tag1consulting.dev.yaml

```diff
@@ -0,0 +1,106 @@
+# Configuration for vagrant-test.tag1consulting.dev
+#
+# This file contains example Hiera configuration that can be used to
+# override or append to default settings in common.yaml.
+# In order to be pulled in by Hiera, the file must be named the same as your VM FQDN
+# (configured in Vagrantfile.local, or defaults to vagrant-test.tag1consulting.dev)
+# with a .yaml extension. e.g. vagrant-test.tag1consulting.dev.yaml
+
+#################################################
+# Apache configuration.
+
+# Example 1: expand the default vhost definition to include an additional
+# <Location> block and an alias to access it.
+#
+# The hostname 'vagrant-test.tag1consulting.dev' matches the definition in common.yaml.
+site_profile::web::vhosts:
```

Comment:
Since we're including a vhost in this hiera file, I think it makes sense to remove it from common.yaml. It was only there to have a vhost setup by default in Vagrant, so I don't see a reason for it if we have a vagrant host specific hiera file.